src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/cimChildJobBase.cs

```diff
@@ -49,11 +49,6 @@ internal CimChildJobBase(CimJobContext jobContext)
             this.Name = this.GetType().Name + _myJobNumber.ToString(CultureInfo.InvariantCulture);
             UsesResultsCollection = true;
 
-            lock (s_globalRandom)
-            {
-                _random = new Random(s_globalRandom.Next());
-            }
-
```

Comment:
I was not 100% sure, but can someone confirm that this pattern is only to make _random thread-safe?

Reply:
@PaulHigin Could you please comment?

Reply:
This pattern is to protect `s_globalRandom.Next()` call from threads in parallel.
The `_random` here is created with a seed, and is guaranteed to use different seed for different instance of `CimChildJobBase`. I don't think this guarantee will be met when switching to `Random.Shared`, so I suggest reverting changes in this file to be safe.